rct229/ruletest_engine/ruletest_jsons/scripts/excel_to_test_json_utilities.py

```diff
@@ -229,7 +229,7 @@ def create_dictionary_from_excel(spreadsheet_name, sheet_name, rule_set):
     invalid_first_keys = ["Notes", "template_lookup"]
 
     # Iterate column by column through values_df
-    for test_id, columnData in tests_df.iteritems():
+    for test_id, columnData in tests_df.items():
```

Comment:
@jugonzal07 This was changed because the ```iteritems``` function was deprecated ([link](https://pandas.pydata.org/pandas-docs/version/1.5/reference/api/pandas.DataFrame.iteritems.html#:~:text=Deprecated%20since%20version%201.5.0%3A%20iteritems%20is%20deprecated%20and%20will%20be%20removed%20in%20a%20future%20version.%20Use%20.items%20instead.)).